test/create-user.test.ts

```diff
@@ -0,0 +1,117 @@
+import * as request from 'supertest';
+import { expect } from 'chai';
+import { User } from '../src/entity/user';
+import * as bcrypt from 'bcrypt';
+
+describe('Mutation createUser', () => {
+  afterEach(async () => {
+    const userRepository = User.getRepository();
+    await userRepository.clear();
+
+    queryCreateUser.variables.data.email = defaultUserTestEmail;
+    queryCreateUser.variables.data.password = defaultUserTestPassword;
+  });
+
+  it('should create user on database and return its info', async () => {
+    const response = await request('localhost:4000').post('/').send(queryCreateUser);
+
+    const userPredefinedData = queryCreateUser.variables.data;
+    const responseUserInfo = response.body.data.createUser;
```

Comment:
Como comentamos lá no doc: o ideal é que não incluamos testes de outras queries/mutations no setup do teste de outras. Justamente porque se cada teste conseguir ser independente de outras queries/mutations, fica mais fácil identificar possíveis bugs no futuro

Reply:
Você está se referindo it('should create user on database and return its info' na linha 15? O request que eu fiz é o da mutation que é pra ser testada mesmo, não é setup. Mas de fato preciso reorganizar esse teste pra ordem ficar mais clara

Reply:
Nvm, vi errado 😅